WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -78,7 +108,31 @@ private extension DashboardViewController {
     }
 
     func configureTitle() {
-        navigationItem.title = ServiceLocator.stores.sessionManager.defaultSite?.name ?? Localization.title
+        navigationItem.title = titleName
+    }
+
+    func configureSubtitle() {
+        if isDefaultSiteNameAvailable {
```

Comment:
I think this would read better if we use the early return pattern.
```swift
guard let !isDefaultSiteNameAvailable {
   return
}

// rest of the code
```

Reply:
Changed it to be more readable!